content/05.heterogeneity.md

```diff
@@ -1,18 +1,20 @@
-These methods not only help in reducing the number of features in various types of data [@doi:10.1016/j.media.2020.101660; @doi:10.1038/ncomms14825], but can also be used to visualize structure or artifacts in the data (e.g. [@doi:10.1038/s41467-019-13056-x]), to define sample subgroups (e.g. [@doi:10.1038/s41467-020-15351-4], or for feature selection and extraction during application of specific machine learning models [@doi:10.1007/978-3-030-03243-2_299-1] (Figure {@fig:1}D).
+Dimensionality reduction methods can help ‘compress’ information from a large number of features into a smaller number of features in an unsupervised manner [@doi:10.1007/978-3-540-33037-0; @doi:10.1098/rsta.2015.0202, @https://www.jmlr.org/papers/v9/vandermaaten08a.html; @https://arxiv.org/abs/1802.03426] (Figure {@fig:1}C).
+An example of a method that is commonly used for dimensionality reduction is principal components analysis (PCA), which identifies new features or dimensions, termed principal components (PCs), that are linear combinations of original features that consecutively capture maximum variance and are uncorrelated with one another [@doi:10.1098/rsta.2015.0202].
+In practice, researchers often use the first few PCs to reduce the dimensionality without removing what may be important or informative variability in the data, though PCs are obtained without regard for labels (e.g., disease vs. control or dataset of origin).
+Beyond reducing the number of features in various types of data [@doi:10.1016/j.media.2020.101660; @doi:10.1038/ncomms14825], dimensionality reduction can also be used to visualize structure or artifacts in the data (e.g., [@doi:10.1038/s41467-019-13056-x]), to define sample subgroups (e.g., [@doi:10.1038/s41467-020-15351-4], or for feature selection and extraction during application of specific machine learning models [@doi:10.1007/978-3-030-03243-2_299-1] (Figure {@fig:1}D). 
 
-Methods like PCA, MDS, t-SNE, and UMAP can successfully identify the effect of these variables on the original data, though t-SNE and UMAP may require tuning of hyperparameters that may effect the output [@https://arxiv.org/abs/1802.03426; @doi:10.23915/distill.00002].
-Furthermore, testing multiple dimensionality reduction methods, rather than a single method, may be necessary to obtain a more comprehensive portrait of the data [@doi:10.1186/s13059-020-02021-3]. 
+Methods like PCA, multidimensional scaling (MDS), t-distributed stochastic neighbor embedding (t-SNE), and uniform manifold approximation and projection (UMAP) can help researchers successfully identify useful patterns in the original data, though t-SNE and UMAP may require tuning of hyperparameters – values used to set up a model that are often specified by a user, rather than learned from the data – that may effect the output [@https://arxiv.org/abs/1802.03426; @doi:10.23915/distill.00002].
```

Comment:
```suggestion
Methods like PCA, multidimensional scaling (MDS), t-distributed stochastic neighbor embedding (t-SNE), and uniform manifold approximation and projection (UMAP) can help researchers successfully identify useful patterns in the original data, though t-SNE and UMAP may require adjusting the hyperparameters – values used to set up a model that can be specified by a user, rather than learned from the data – that may effect the output [@https://arxiv.org/abs/1802.03426; @doi:10.23915/distill.00002].
```

Reply:
Tuning hyperparameters sounds intimidating, thought maybe this was more friendly?